hypha/apply/api/v1/views.py

```diff
@@ -61,6 +61,19 @@ def get_queryset(self):
             Prefetch('reviews', Review.objects.submitted()),
         )
 
+    @action(detail=True, methods=['put'])
+    def set_summary(self, request, pk=None):
+        applicationSubmission = self.get_object()
+        applicationSubmission.summary = request.data.get("summary")
+        serializer = self.get_serializer(
+            applicationSubmission, data=request.data, partial=True)
+        if serializer.is_valid():
```

Comment:
`serializer.is_valid(raise_exception=True)` would be good instead of the if/else condition.

ref: [1](https://stackoverflow.com/a/40373171), [2](https://www.django-rest-framework.org/api-guide/serializers/#raising-an-exception-on-invalid-data)